IntelliTect.TestTools.Console.Tests/ConsoleAssertTests.cs

```diff
@@ -91,29 +91,21 @@ public void ConsoleTester_HelloWorld_MissingNewline()
         [TestMethod]
         public void ExecuteProcess_PingLocalhost_Success()
         {
-            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                Assert.Inconclusive("Platforms other than windows have not been tested.");
 
-                /* Try the following:
-                 * Command: ping.exe -c 4 localhost                 (the '-c 4' limits the ping to 4 times)
-                 * PING localhost (104.24.0.68) 56(84) bytes of data.
-                 *  64 bytes from 104.24.0.68: icmp_seq=1 ttl=58 time=47.3 ms
-                 *  64 bytes from 104.24.0.68: icmp_seq=2 ttl=58 time=51.9 ms
-                 *  64 bytes from 104.24.0.68: icmp_seq=3 ttl=58 time=57.4 ms
-                 *  64 bytes from 104.24.0.68: icmp_seq=4 ttl=58 time=57.4 ms
-                 */
+            string expected = $@"*
+Pinging * ?::1? with 32 bytes of data:
+Reply from ::1: time*";
 
-            }
-            else
+            string pingArgs = "-c 4 localhost";
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                ConsoleAssert.ExecuteProcess(
-$@"
-Pinging * ?::1? with 32 bytes of data:
-Reply from ::1: time*",
-                "ping.exe", "-n 4 localhost", out string standardOutput, out _);
-                Assert.IsTrue(standardOutput.ToLower().IsLike($"*{ Environment.MachineName.ToLower()}*"));
+                pingArgs = pingArgs.Replace("-c ", "-n ");
             }
+
+            ConsoleAssert.ExecuteProcess(
+                expected,
+                "ping.exe", pingArgs, out string standardOutput, out _);
```

Comment:
The problem here, is just because a tool exists on one linux distribution, doesn't guarantee that it will exist on any others. For example, Alpine linux (popular with many container like things) is pretty bare bones and has almost nothing by default. While something like ubuntu likely has it by default. Testing in WSL is fine, but you have to remember that they put in a lot of magic to forward to windows commands (which is why I suspect it worked for you).

As for making this particular example work (if we assume the system has `ping` installed). I suspect if you just drop the ".exe" portion it will likely work.